modules/users/server/routes/admin.server.routes.js

```diff
@@ -4,22 +4,28 @@
  * Module dependencies
  */
 var adminPolicy = require('../policies/admin.server.policy'),
-  admin = require('../controllers/admin.server.controller');
+  admin = require('../controllers/admin.server.controller'),
+  passport = require('passport'),
+  express = require('express');
 
 module.exports = function (app) {
-  // User route registration first. Ref: #713
-  require('./users.server.routes.js')(app);
+  var router = express.Router();
 
-  // Users collection routes
-  app.route('/api/users')
+  //Set JWT Auth for all Admin Routes
+  router.all('*', passport.authenticate('jwt', { session: false }));
+
+  router.route('/')
     .get(adminPolicy.isAllowed, admin.list);
 
   // Single user routes
-  app.route('/api/users/:userId')
+  router.route('/:userId')
     .get(adminPolicy.isAllowed, admin.read)
     .put(adminPolicy.isAllowed, admin.update)
     .delete(adminPolicy.isAllowed, admin.delete);
 
   // Finish by binding the user middleware
-  app.param('userId', admin.userByID);
+  router.param('userId', admin.userByID);
+
+  app.use('/api/admin/users', router);
```

Comment:
@mleanos I think I found the best of both worlds.  If i regex the admin :userId route it will work on the same base route since things like /me and /password won't get caught:

router.route('/:userId([0-9a-fA-F]{24})')

Need a contributor to weigh in.  @codydaig @ilanbiala would you rather see a new base path /api/admin/users or a regex on the route?